apisix/core/id.lua

```diff
@@ -20,17 +20,23 @@
 -- @module core.id
 
 local fetch_local_conf = require("apisix.core.config_local").local_conf
-local try_read_attr    = require("apisix.core.table").try_read_attr
-local log              = require("apisix.core.log")
-local uuid             = require('resty.jit-uuid')
-local smatch           = string.match
-local open             = io.open
-
-
+local try_read_attr = require("apisix.core.table").try_read_attr
```

Comment:
please maintain code beautification (equals on top of each other)

Reply:
done